frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "@emotion/styled";
+import { DropTargetMonitor, useDrop } from "react-dnd";
+
+import { PossibleDroppableObject } from "../../ui-components/Dropzone";
+
+interface Props<DroppableObject> {
+  onDrop: (item: DroppableObject, monitor: DropTargetMonitor) => void;
+  acceptedDropTypes: string[];
+  lastElement?: boolean;
+}
+
+const RootHeightBase = 30;
+const LineHeight = 3;
+
+const Root = styled("div")`
+  width: 100%;
+  left: 0;
+  right: 0;
+  position: relative;
+  border-radius: ${({ theme }) => theme.borderRadius};
+`;
+
+const Line = styled("div")`
+  overflow: hidden;
+  display: block;
+  background-color: ${({ theme }) => theme.col.blueGrayLight};
+  margin: 1px 0;
+  height: ${LineHeight}px;
+  border-radius: 2px;
+`;
+
+const BetweenElements = <DroppableObject extends PossibleDroppableObject>({
+  acceptedDropTypes,
+  onDrop,
+  lastElement,
+}: Props<DroppableObject>) => {
+  const [{ isOver }, addZoneRef] = useDrop({
+    accept: acceptedDropTypes,
+    drop: onDrop,
+    collect: (monitor) => ({
+      isOver: monitor.isOver(),
+      isDroppable: monitor.canDrop(),
+    }),
+  });
+
+  const rootHeightMultiplier = lastElement ? 0.5 : 1;
+  const rootDefaultMarginTop = (lastElement ? -15 : -5) - LineHeight;
+  const rootOverMarginTop = lastElement ? -23 : -10;
+  const rootDefaultTop = lastElement ? -5 : -10;
+  const rootOverTop = (lastElement ? -2 : -15) - LineHeight;
```

Comment:
Those calculations seem a bit arbitrary with a lot of magic numbers. Are they necessary? Why does the height and marginTop need to change? Aren't the lines visually always the same height?

Reply:
The height needs to change so that the elements below do not bounce up and down. This affects the margin as well => we need to take away less space than before. 

I was able to take one of the magic values out and simplify some more. I think the other magic values are needed.

Reply:
I don't think `absolute` would help us in this case, since we would need to find the position for the root by a lot more complex calculations (e.g. scroll positions etc.). I think `relative`is the intended display property.

Reply:
When you say `absolute`, are you referring to `position: fixed`? Fixed would mean related to the viewport / scroll positions - like you say, but `absolute` simply relates to the parent container that would have to be `relative` (within DropzoneList)

Reply:
Sorry, you are right. That would work. I will try out if that works better.